fabric-chaincode-shim/src/main/java/org/hyperledger/fabric/shim/impl/ChaincodeSupportClient.java

```diff
@@ -112,6 +113,29 @@ public void onCompleted() {
 
         );
 
+        try {
+            start(itm, requestObserver);
```

Comment:
I'd like to try and see if we can unify these start methods; they are very similar.

Once the request observers have been setup, the rest of the logic is the same.